source/PlayState.hx

```diff
@@ -49,7 +62,37 @@ class PlayState extends FlxState
 	override public function update(elapsed:Float):Void
 	{
 		super.update(elapsed);
-		hud.update(elapsed);
+
 		FlxG.collide(map, player);
+
+		// If platform and player are not touching, allow offset to be overwritten
+		if (!FlxG.overlap(player, platform)) {
+			platform.sticky = false;
+		}
+
+		if (FlxG.collide(player, platform)) {
+			// Only set offset value if touching platform for "first" time
+			player.acceleration.y = 8000;
```

Comment:
I'd rather have this number stored in a variable in the Platforms class, if possible.